cfg/config.go

```diff
@@ -3,11 +3,89 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"errors"
+	"log"
+	"os"
+	"strings"
+)
+
+// Load configs from a env file & sets them in environment variables
+func loadEnvVar() error {
+	f, err := os.Open(".env")
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			log.Printf("%s", err)
+		}
+	}()
+
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(f)
+
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		return err
+	}
+
+	for _, l := range lines {
+		pair := strings.Split(l, "=")
+		if len(pair) != 2 {
+			return errors.New("not enough data for the configuration in .env file")
+		}
+		os.Setenv(pair[0], pair[1])
```

Comment:
@iamRosty  It is desirable to separate with an empty line the rows 43 and 44

Reply:
Thank you, fixed.